devops/providers/azure-devops/templates/infrastructure/azure-pipeline-build-stage.yml

```diff
@@ -1,11 +1,13 @@
 parameters:
+  alwaysRun: 'false'
```

Comment:
A thought, but not necessarially something to change. Does it make sense to (additionally? instead of?) drive this by an env variable so that it can be over-run on a per-deployment basis w/o a code change?

Reply:
Great suggestion -- will change to an environment variable.